### ISSUE TITLE
Dark Mode - Low Contrast Selected Characters in Character Map

### ISSUE THREAD

**Issue by WatchMkr** [pop-os/gtk-theme#328]
Also check Character Map in light mode.

**Comment by isantop** [comment]
https://gitlab.gnome.org/GNOME/gucharmap/blob/master/gucharmap/gucharmap-chartable.c#L904

It appears the charmap_chartable widget being used here is using custom Cairo drawing, and while it gets the current theme selected background-color, it doesn't seem to get the foreground color, which is causing this issue. The same issue is present in Yaru and Adwaita, however those themes use different selection background colors.

Yaru does actually exhibit this issue in the Light theme, where the dark orange selection has very poor contrast with the dark text:

![Screenshot from 2019-10-14 10-18-18](https://user-images.githubusercontent.com/5883565/66767257-3801d180-ee6d-11e9-98e4-e81a636112db.png)

(Background: #E45431, Foreground: #3F3F3F, Contrast Ratio: 2.8:1)

**Comment by jackpot51** [comment]
@WatchMkr please confirm that the issue still exists

**Comment by isantop** [comment]
Nope, sorry, that was the wrong issue. This one is definitely there
